mathesar_ui/src/pages/record/TableWidget.svelte

```diff
@@ -46,15 +46,19 @@
   </div>
 </div>
 
-<style>
+<style lang="scss">
   .top {
     display: grid;
     grid-template: auto / 1fr auto;
     gap: 0.5rem;
     justify-content: space-between;
     align-items: center;
-    padding-bottom: 0.5rem;
+
     overflow: hidden;
+    > h3 {
+      font-size: var(--text-size-large);
```

Comment:
I would rather us define this sort of heading tyography CSS globally instead of per-component.